lib/protocol-decoder.js

```diff
@@ -0,0 +1,84 @@
+'use strict'
+
+var proxyProtocol = require('proxy-protocol-js')
+
+function protocolDecoder (client, data) {
+  var proto = {}
+  // var buffer = Buffer.allocUnsafe(0)
+  // buffer = client.conn.read(null)
+  if (!data) return proto
+  // todo: checkProxiesList(client.conn, client.broker.trustedProxies)
```

Comment:
What is this todo?

Reply:
Just a reminder to talk about another feature, this one would provide another hook to check server ( proxy ) address and compare it with a proxy whitelists / blacklists ( if the list and the callback are defined ).

Reply:
Would you mind removing this todo and opening an issue instead?